app/views/shelters/show.html.erb

```diff
@@ -9,4 +9,13 @@
     <p>Zip: <%= @shelter.zip %></p>
     <p><%= link_to("View this shelter's pets", "/shelters/#{@shelter.id}/pets", class: "name") %></p>
   </div>
+  <%= link_to 'New Review', "/shelters/#{@shelter.id}/new_review" %>
```

Comment:
note to self: instance variables returning null will act like nothing in the URI